deploy/quickstart.sh

```diff
@@ -31,7 +32,7 @@ oc create secret generic elasticsearch \
     --from-file=logging-es.crt \
     --from-file=admin-key=system.admin.key \
     --from-file=admin-cert=system.admin.crt \
-    --from-file=admin-ca=ca.crt
+    --from-file=admin-ca=ca.crt || true
```

Comment:
Can't comment down there, but L51 will also need the `oc create ... | oc apply` technique if it is to be re-run. I haven't checked the deploy script itself.